.changeset/mean-deers-confess.md

```diff
@@ -0,0 +1,5 @@
+---
+'jest-emotion': major
+---
+
+Added the `T` parameter to the `Matchers` interface in the typescript definitions for jest-emotion
```

Comment:
```suggestion
Added the `T` parameter to the `Matchers` interface in the TypeScript definitions to make this module compatible with `@types/jest@^24.0.20`.
```

Reply:
I've wanted to commit this suggestion through GitHub interface - but seems like I don't have sufficient permissions to push to your fork.

You can check how to allow this for the future PRs here - https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork . But at the moment just commit this suggestion with a click on a button.